### ISSUE TITLE
Should parentheses be banned for conditions, even for ternaries?

### ISSUE THREAD

**Issue by tobypinder** [rubocop/rubocop#364]
> Don't use parentheses around the condition of an if/unless/while/until

Throws for the following snippet.

``` ruby
        data: {
          confirm: (method == :delete) ? 'Are you sure you wish to delete this?' : nil,
          icon: icon_parse(icon)
        },
```

I personally realise that this is all subjective (hence Cops are modular) but should the **ParenthesesAroundCondition** cop really be covering ternaries like this? The following personally seems a bit less readable.

``` ruby
        data: {
          confirm: method == :delete ? 'Are you sure you wish to delete this?' : nil,
          icon: icon_parse(icon)
        },
```

It might be a good idea to have a **ParenthesesAroundTernaryCondition** or whatever. I appreciate any and all feedback on whether there would be demand for such separation vs the ease of configuration and reduced complexity of a smaller number of cops.
